roles/sap_general_preconfigure/meta/argument_specs.yml

```diff
@@ -0,0 +1,299 @@
+        description:
+          - The list of packages to install.
+          - The default for this variable is set in the vars file which corresponds to the detected OS version.
+        required: false
+        type: list
+        elements: str
+
+      sap_general_preconfigure_min_package_check:
+        default: true
+        description:
+          - The default is to install or check if the minimum package versions are installed as defined in the vars files.
+          - Set to `false` if you do not install or check these minimum package versions.
+        required: false
+        type: bool
+
+      sap_general_preconfigure_update:
+        default: false
+        description:
+          - By default, the role will not update the system, for avoiding an unintentional minor OS release update.
+          - Set this parameter to `true` if you want to update your system to the latest package versions.
+          - When using SAP HANA, make sure to set the release lock properly so the minor OS release will be one of
+          - those for which SAP HANA is supported. See also `sap_general_preconfigure_set_minor_release`.
+        required: false
+        type: bool
+
+      sap_general_preconfigure_reboot_ok:
+        default: false
+        description:
+          - Set to `true` if you want to perform a reboot at the end of the role, if necessary.
+        required: false
+        type: bool
+
+      sap_general_preconfigure_fail_if_reboot_required:
+        default: true
+        description:
+          - If `sap_general_preconfigure_reboot_ok` is set to `false`, which is the default, a reboot requirement should not
+          - remain unnoticed. For this reason, we let the role fail. Set this parameter to `false` to override this behavior.
+          - Can be useful if you want to implement your own reboot handling.
+        required: false
+        type: bool
+
+      sap_general_preconfigure_selinux_state:
+        default: 'permissive'
+        description:
+          - One of the SELinux states to be set on the system.
+        choices:
+          - 'enforcing'
+          - 'permissive'
+          - 'disabled'
+        required: false
+        type: str
+
+      sap_general_preconfigure_modify_selinux_labels:
+        default: true
+        description:
+          - Set to `false` if you do not want to modify the SELinux labels for the SAP directory `/usr/sap`.
+        required: false
+        type: bool
+
+      sap_general_preconfigure_size_of_tmpfs_gb:
+        default: "{{ ((0.75 * (ansible_memtotal_mb + ansible_swaptotal_mb)) / 1024) | round | int }}"
+        description:
+          - The size of the tmpfs in GB. The formula used here is mentioned in SAP note 941735.
+        required: false
+        type: str
+
+      sap_general_preconfigure_modify_etc_hosts:
+        default: true
+        description:
+          - Set to `no` if you do not want the role to modify the `/etc/hosts` file.
```

Comment:
I assume this should be "Set to `false`..." for consistency?

Reply:
Thanks, makes sense!